src/lib/utils/platform.js

```diff
@@ -39,9 +38,11 @@ export const isIOS = isIOSWeb || isIOSNative
 
 export const isAndroid = isAndroidWeb || isAndroidNative
 
+export const isBrowser = !isMobile
```

Comment:
```suggestion
// if Platform.OS is 'web' (e.g. running on web), will return isBrowser flag from the device detect library. 
// otherwise (e.g. running on native) will return false (because library wrongly returns true in that case)
export const isBrowser = isWeb ? isBrowserWeb : false
```